src/index/ivf/ivf.cc

```diff
@@ -498,14 +509,32 @@ IvfIndexNode<T>::RangeSearch(const DataSet& dataset, const Config& cfg, const Bi
                     auto cur_data = (const uint8_t*)xq + index * dim / 8;
                     index_->range_search_thread_safe(1, cur_data, radius, &res, index_->nlist, bitset);
                 } else if constexpr (std::is_same<T, faiss::IndexIVFFlat>::value) {
-                    auto cur_data = (const float*)xq + index * dim;
-                    index_->range_search_without_codes_thread_safe(1, cur_data, radius, &res, index_->nlist, 0, bitset);
+                    auto cur_query = (const float*)xq + index * dim;
```

Comment:
Why we don't do 
```
auto cur_query = (const float*)xq + index * dim;
if (is_cosine) {
  cur_query = CopyAndNormalizeFloatVec(cur_query, dim);
}
```
And then we don't have to do any changes in the if else blocks

Reply:
updated